### PR TITLE
Linking note pad object

### PR DESCRIPTION
**Pull request by becc-mu**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+let notePad;
 window.addEventListener("load", function() {
-  document.getElementById("app").innerHTML = Index().render();
+  notePad = new NotePad;
+  document.getElementById("app").innerHTML = IndexPage().render(notPad.notes);
```
Comment (shannongamby) [inline app.js]:
```suggestion
  document.getElementById("app").innerHTML = IndexPage().render(notePad.notes);
```

Comment (shannongamby) [reply app.js]: Also, I don't know if 'n' needs to be a capital too

Comment (shannongamby) [reply app.js]: No it doesn't, but yeah, missing 'e'

Comment (shannongamby) [reply app.js]: I changed it - hope that's ok :)

Comment (toddpla) [reply app.js]: good spot

**Pull request merged**